src/celeritas/ext/RootJsonDumper.cc

```diff
@@ -0,0 +1,46 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/ext/RootJsonDumper.cc
+//---------------------------------------------------------------------------//
+#include "RootJsonDumper.hh"
+
+#include <ostream>
+#include <TBufferJSON.h>
+#include <TClass.h>
+
+#include "corecel/Assert.hh"
+#include "corecel/io/Logger.hh"
+#include "celeritas/io/ImportData.hh"
+
+#include "RootFileManager.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Construct from path to ROOT file.
```

Comment:
```suggestion
 Construct with output stream object.
```

Likely a remnant copy-paste from RootImporter. Same for the `operator()` below.

Reply:
Sorry about not reviewing this more thoroughly before sending it on to you! Dumb copy paste errors.

Reply:
Don't worry. Happy to try and catch these.